draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2883,7 +2889,125 @@ https://video.example.com/k/movie1/4/013.ts
          ]]></artwork>
           </figure>
         </section>
-      </section>
+	</section>
+	<section anchor="execution-policy" title="Execution Extension">
+	<t>
+		Unless specified otherwise, dCDN is at liberty to decide how to choose trigger commands for execution from all pending commands, whether to process trigger commands sequentially or in parallel, immediately upon acceptance, or with a delay in batches. A uCDN may wish to control trigger processing in more detail, including order of execution, dependencies, and concurrency.
```

Comment:
I will note that we really haven't provided guidance regarding how TimePolicy should interact with ExecutionPolicy.  I think you have mentioned previously how TimePolicy should govern the time window when the trigger can execute, while the priority of ExecutionPolicy would govern the order in which similar triggers would execute.  Additionally, it seems like there are some potential error conditions to call out, such as:
1. When a trigger is marked Urgent, but TimePolicy prevents immediate execution
2. When a trigger with a TimePolicy in the present has a dependency on another trigger that is scheduled for a TimePolicy in the future
It seems that the dCDN should reject such cases.

Reply:
Added language